app/javascript/components/explore/ExploreDisplayTabs.js

```diff
@@ -441,7 +459,12 @@ export function getEnabledTabs(exploreInfo, exploreParams) {
         enabledTabs = ['dotplot', 'heatmap']
       }
     } else {
-      enabledTabs = ['scatter', 'distribution']
+      if (exploreParams.annotation.type === 'numeric') {
```

Comment:
I'm going to renew my request that we look at this method and try to figure out a way to simplify the signature.  We're up to 9 different plot/tab types, and this method keeps getting more byzantine as we move forward.

Reply:
agree that we haven't found the optimal form yet.

Reply:
agree that we haven't found the optimal form for this yet.